### PR TITLE
MKT-872 -- Pricing Page v2

### PR DESCRIPTION
**Pull request by stewsmith**
https://optimizely.atlassian.net/browse/MKT-872

### CODE REVIEW

**Comment by teeyoutan** [comment]
@stewsmith  its failing so I can't preview it?

**Comment by stewsmith** [comment]
retest this please

**Comment by KarenLoughrey** [comment]
@stewsmith @teeyoutan Thanks! Translations are now in progress, will keep you posted and let you know as soon as they're finalised.

**Comment by dtothefp** [comment]
@stewsmith make the hipster check boxes 

screen shot 2015-02-05 at 4 57 51 pm

**Comment by dtothefp** [comment]
in media query center the triangle arrow 

screen shot 2015-02-05 at 4 58 57 pm

**Comment by dtothefp** [comment]
@stewsmith makes these buttons a consitent width in mobile, I usually use 90% 

screen shot 2015-02-05 at 5 06 38 pm

**Comment by dtothefp** [comment]
@stewsmith should the phone number at the top be clickable?

**Comment by dtothefp** [comment]
@stewsmith make buttons consistent width in mobile on `features-and-plans`

screen shot 2015-02-05 at 5 11 47 pm

**Comment by dtothefp** [comment]
@stewsmith make the margins here the same as the bullet list above

screen shot 2015-02-05 at 5 12 38 pm

**Comment by dtothefp** [comment]
@stewsmith fix me :sob: 

screen shot 2015-02-05 at 5 13 14 pm

**Comment by dtothefp** [comment]
@stewsmith buttons at bottom don't work on `feature-list`. Also, don't forget to adjust the CSS here for the new responsive design we came up with

File: website-guts/assets/js/pages/features-and-plans.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+$('#talk-to-us').click(function(e){
```
Comment (dtothefp) [inline website-guts/assets/js/pages/features-and-plans.js]: @stewsmith I think we should be using `.on` for all events as it is more extensible

**Comment by stewsmith** [comment]
@dtothefp Your comments are still on my TODO. I'm merging this to test the flow of the modals on production.

**Pull request merged**